### ISSUE TITLE
Multiple digit components confuses findSequenceOnDisk()

### ISSUE THREAD

**Issue by justinfx** [justinfx/fileseq#76]
Repro:

```
mkdir test_seq && cd test_seq 
loop -t 1-9 "touch foofile_#.0001.exr"
```

```
fileseq.findSequenceOnDisk('/tmp/test_seq/foofile_#.0001.exr')

filesequence.py in findSequenceOnDisk(cls, pattern, strictPadding)
    699 
    700         msg = 'no sequence found on disk matching {0}'
--> 701         raise FileSeqException(msg.format(pattern))
    702 
    703     @classmethod

FileSeqException: no sequence found on disk matching /tmp/test_seq/foofile_#.0001.exr
```

**Comment by justinfx** [comment]
The problem occurs within `FileSequence.yield_sequences_in_list()`.

1. First the original pattern if given to `findSequenceOnDisk`, which parses and globs for the files on the filesystem. This returns the correct list of individual files.
1. `yield_sequences_in_list` is then passed this globbed list of files to try and turn them into sequences.

The problem is that when `yield_sequences_in_list` receives the file list, it will get something like `seq/file_0003.0001.exr`
for the original pattern having been `seq/file_#.0001.exr`
and now there is a loss of context. It is ambigous to `yield_sequences_in_list` where the real frame pattern is and the `constants.DISK_RE` regex is parsing a basename of `file_0003` and an ext of `.0001.exr`.

So it isn't a simple matter of improving a regex to solve this problem. The parent `findSequenceOnDisk` knows the correct basename, frame pattern position, and ext, and there needs to be some way to use that context in the `yield_sequences_in_list` call.

**Comment by justinfx** [comment]
Fixed via #78
